### PR TITLE
Adding jest to website project

### PR DESCRIPTION
**Pull request by johnrjj**
Tested with a mock test, everything works. I will add some tests for reference shortly, but wanted to add the core infra in a standalone PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
         "build:prod": "node --max_old_space_size=16384 ./node_modules/webpack/bin/webpack.js --mode production",
         "build:dev": "node --max-old-space-size=16384 ./node_modules/webpack/bin/webpack.js --mode development",
         "clean": "shx rm -rf public/bundle* mdx/tools",
+        "test": "jest --passWithNoTests",
```
Comment (johnrjj) [inline package.json:14]: Will remove `--passWithNoTests` in the next PR that adds tests.

Comment (kimpers) [inline package.json:14]: could you add this to the `pre_push` command as well?

**Review by kimpers — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (johnrjj) [reply package.json:14]: yup!

Comment (johnrjj) [reply package.json:14]: added

**Pull request merged**